src/app/components/ConnectorForm/index.tsx

```diff
@@ -70,28 +70,27 @@ function ConnectorForm({
             title
           )}
         </div>
-        {media}
         {description && (
-          <div className="text-gray-500 dark:text-white whitespace-pre-line">
+          <div className="text-gray-700 dark:text-white whitespace-pre-line mb-8">
```

Comment:
I think it would be cleaner to use gap / space-y classes on the container instead of adding those margins on every child element so the gap can be adjusted at one place.